dev_tools/scripts/format_header.sh

```diff
@@ -266,9 +266,6 @@ fi
 # Arrange the remain files and give
 if [ -f "${CONTENT}" ]; then
     add_regroup
-    if [ "${HAS_HIP_RUNTIME}" = "true" ]; then
```

Comment:
What is the reason for these changes again? Do we no longer need it?

Reply:
the behavior still exists. DPCPP gives more additional header than <hip/runtime.h>, so I move the all additional before the LICENSE into header section not just hip/runtime.h